sub/source/docs/arming_the_motors.rst

```diff
@@ -0,0 +1,29 @@
+.. _arming_the_motors:
+
+=================
+Arming the motors
+=================
+
+Arming the vehicle allows the thrusters to start spinning.  Before arming,
+make sure all people, objects, and any body parts (e.g., hands) are clear of the thruster propellers.
+
+In order to arm:
+
+- Pre-Arm safety checks must be passing (See :ref:`common-prearm-safety-checks`). It is recommended that :ref:`ARMING_CHECK<ARMING_CHECK>` be enabled for all available systems/sensors checks to prevent arming with system issues present. You must have also provided a method to Disarm the vehicle, either by GCS button assignments (``BTNx_FUNCTION``), or :ref:`AUX_FUNCTION <common-auxiliary-function>`.
+- The throttle must be at idle value if :ref:`RC_OPTIONS<RC_OPTIONS>` bit 5 is set ("Arming check throttle for 0 input"). Normally, this would be a center stick if bi-directionaa vertical thrusters are used.
+- An arming pilot control must be activated, eg. GCS joystick button with "Arm" function assigned, GCS MAVLink ARM command given, or an :ref:`Auxiliary function <common-auxiliary-functions>` such as ARM/DISARM ("153") is used by either by GCS command or RC channel switch (if enabled).
+
+Disarming the motors
+====================
+
+Disarming the motors will prevent the motors from spinning. The motors are disarmed when:
```

Comment:
It may be worth including some troubleshooting information around:

- output trim values, which can be required to enter the ESC's neutral region, due to either a poor quality clock causing a timing mismatch, or electrical signal smoothing due to either parasitic capacitance or low voltage (e.g. from long signal wires to the ESC)
- non-bidirectional ESCs, which typically expect a 1000µs pulse-duration for the stop/neutral signal, rather than the 1500µs pulses expected by the autopilot

Reply:
already covered on our std ESC cal page which is already in the Sub wiki

Reply:
Seems ok to cover in a separate page, but [the ESC calibration page](https://ardupilot.org/sub/docs/common-esc-calibration.html) only mentions trim in the context of brushed motors, and more generally specifies that ESCs can be calibrated using `SERVOn_MIN/MAX` params, without mentioning the group-level `MOT_PWM_MIN/MAX`. If I'm remembering correctly, `SERVOn_MIN/MAX` are currently ignored for multicopter motor outputs, but it's at least not made clear how those separate limits interract.

That said, that's not key information for this PR, just a flagging of some potentially outdated docs (which I can try to look into when I get back to focusing on ArduSub docs, hopefully next week).

Reply:
yes, page needs a little tweaking....will put it on the list